opm/material/components/co2tables.inc

```diff
@@ -1,3 +1,6 @@
+#ifndef CO2TABLES_INC
```

Comment:
Maybe it would be good if the numbers below was taken from a pure inputfile or at least as an option from an inputfile.

Reply:
The CO2 table should cover the range of interest, so I don't see the point of making it an input file. On the contrary the point of the CO2STORE option is to give the user an option of not setting up the PVT input tables themselves. If the user what to use its own table he can always use the PVTO etc. options.